librz/core/cmd/cmd_print.c

```diff
@@ -3966,6 +3967,46 @@ RZ_IPI RzCmdStatus rz_cmd_disassembly_function_handler(RzCore *core, int argc, c
 	return RZ_CMD_STATUS_OK;
 }
 
+RZ_IPI RzCmdStatus rz_print_function_rzil_handler(RzCore *core, int argc, const char **argv) {
+	ut64 start, end, size, oldoff;
+	RzAnalysisFunction *fcn;
+	RzList *list;
```

Comment:
You can move those declarations where they are used. Reduces line count and it is easier to see what happens down below.

Reply:
there is really no reason. but they should be initialized at most to 0.